### ISSUE TITLE
False Positive UnnecessaryApply

### ISSUE THREAD

**Issue by scottkennedy** [arturbosch/detekt#1473]
## Expected Behavior
No UnnecessaryApply

## Observed Behavior
UnnecessaryApply

## Steps to Reproduce
```kt
    fun detektFalsePositive() {
        Preference(context).apply { // This one does not trigger
            summary = "test"
            isEnabled = false
        }

        Preference(context).apply { // This triggers UnnecessaryApply
            if (shouldDisable()) {
                summary = "test"
                isEnabled = false
            }
        }
    }
```

In my project the Android Preference is the return value from another function, so I can't put the if statement around the whole thing.

## Your Environment
* Version of detekt used: RC13
* Version of Gradle used (if applicable): 5.2.1
* Operating System and version: Various
* Link to your project (if it's a public repository):
